web-admin/src/features/organizations/hibernating/wakeAllProjects.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  adminServiceListProjectsForOrganization,
+  createAdminServiceRedeployProject,
+  getAdminServiceGetProjectQueryKey,
+  getAdminServiceListProjectsForOrganizationQueryKey,
+  type V1Project,
+  type V1RedeployProjectResponse,
+} from "@rilldata/web-admin/client";
+import { queryClient } from "@rilldata/web-common/lib/svelte-query/globalQueryClient";
+import { get } from "svelte/store";
+
+export async function wakeAllProjects(organization: string) {
+  const projectDeployer = createAdminServiceRedeployProject({
+    mutation: {
+      queryClient,
+    },
+  });
+  const promises: Promise<V1RedeployProjectResponse>[] = [];
+
+  let pageToken: string | undefined = undefined;
+  do {
+    try {
+      const projectsResp = await queryClient.fetchQuery({
+        queryKey: getAdminServiceListProjectsForOrganizationQueryKey(
+          organization,
+          {
+            pageToken,
+          },
+        ),
+        queryFn: () =>
+          adminServiceListProjectsForOrganization(organization, {
+            pageToken,
+          }),
+      });
+      if (projectsResp.projects.length === 0) break;
+
+      projectsResp.projects.forEach((project) => {
+        if (project.prodDeploymentId) return;
+        promises.push(redeployProject(organization, project, projectDeployer));
+      });
+      pageToken = projectsResp.nextPageToken;
+    } catch {
+      // TODO
+      break;
+    }
+  } while (pageToken);
+
+  try {
+    await Promise.all(promises);
+  } catch {
+    // TODO
+  }
+
+  void queryClient.refetchQueries(
+    getAdminServiceListProjectsForOrganizationQueryKey(organization),
+  );
+}
+
+async function redeployProject(
+  organization: string,
+  project: V1Project,
+  projectDeployer: ReturnType<typeof createAdminServiceRedeployProject>,
+) {
+  const resp = await get(projectDeployer).mutateAsync({
+    organization,
+    project: project.name,
+  });
+  void queryClient.refetchQueries(
+    getAdminServiceGetProjectQueryKey(organization, project.name),
+    {
+      // avoid refetching createAdminServiceGetProjectWithBearerToken
+      exact: true,
+    },
+  );
```

Comment:
Typically, `invalidateQueries` would be best practice to avoid unneeded refetches. (But the QueryObserver instance needs `refetchOnMount: true` due to [this outstanding "gotcha"](https://github.com/rilldata/rill-private-issues/issues/719).)